src/components/Notification/NotificationBase.jsx

```diff
@@ -5,7 +5,7 @@ import { Check, X } from 'react-feather'
 
 import styles from './NotificationBase.module.scss'
 
-const NotificationBase = ({ error, title, onClose }) => {
+const NotificationBase = ({ error, title, withoutIcon = false, onClose }) => {
```

Comment:
Bool props are often clearer to parse for the brain with a "positive" wording, to avoid double negative expressions like here and `!` operators everywhere.
In practice, this means avoiding negative names like `isFooDisabled`, `withoutFoo`, etc. and either reverse the logic with the positive counterparts (`isFooEnabled`, `withFoo`, etc.) or use an action verb so it's easier to read (`omitFoo`).